Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -406,7 +411,7 @@ def audit_deps
           problem "Dependency '#{dep.name}' was renamed; use new name '#{dep_f.name}'."
         end
 
-        if @@aliases.include?(dep.name)
+        if @@aliases.include?(dep.name) && !dep.name.include?("@")
```

Comment:
Same comment as other PR. I think this should probably be encapsulated in formula.rb behind a method like `is_version_formula?` or `is_versioned_formula?`